src/main/webapp/election-info.js

```diff
@@ -200,18 +220,25 @@ function initializeMap() {
           type = "Standard polling location.";
         }
 
-        const description =
+        var description =
```

Comment:
I should have caught this earlier, but is there any reason why we can't use handlebars template here?

Reply:
The reason I didn't use Handlebars here is 1) the map is outside of the Handlebars template and doesn't load correctly within it and 2) this text is for info windows that appear when you click on the map, so it's harder to use Handlebars to set aspects of the map than it is to display the info cards for candidates, etc.

Reply:
I meant just using Handlebars to fill in the blanks in the html within `description` var.

Isn't this just static html after Handlebars fills in the blanks though? I'm confused why it involves setting aspects of the map.